src/arb/acb.jl

```diff
@@ -1281,6 +1282,29 @@ function moddelta(x::acb)
    return z
 end
 
+@doc Markdown.doc"""
+    modeisenstein(x::acb, k::Int)
+
+Return the non-normalized Eisenstein series $G_k(\tau)$ of
+$\mathrm{SL}_2(\mathbb{Z})$. Also defined for $\tau = i \infty$.
+"""
+function modeisenstein(x::acb, k::Int)
```

Comment:
Since these are not normalized, how about the name `modular_eisenstein_g`? Also, since it is written `G_k(x)`, the integer parameter `k` should be first: `modular_eisenstein_g(k::Int, x::acb)`.

Reply:
I do not like the ending `_g`. But `modular_eisenstein_nonnormalized` seems to be far to long, right?

Reply:
I don't like the name `modular_eisenstein_nonnormalized`. My only reasons for suggesting the `_g` was that:
1. If someone ever wants to add in the normalized functions, they could do `modular_eisenstein_e`.
2. If someone ever wants to add functions for other groups, the `_g` would be a hint here that these are the classical G_k.
You can ignore the trailing `_g` if you want.